lib/search.js

```diff
@@ -71,9 +71,11 @@ function search (args, cb) {
     parseable: npm.config.get('parseable'),
     color: npm.color
   })
+  var searchResults = '';
   outputStream.on('data', function (chunk) {
     if (!anyOutput) { anyOutput = true }
     output(chunk.toString('utf8'))
+	searchResults += chunk.toString('utf8')
```

Comment:
Going to want to append a newline to this. `output` is a `console.log`, which has newlines.